extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -208,10 +208,20 @@ describes.realWin(
       doc.body.appendChild(element);
 
       await getRenderedData();
+      let options = fetchJsonStub.getCall(0).args[2];
+      // verify initial call to `batchFetchJsonFor`
+      expect(options.refresh).to.be.false;
+      expect(options.xssiPrefix).to.be.null;
+      expect(options.expr).to.be.equal('.');
```

Comment:
nit, here and elsewhere:
```suggestion
      expect(options.expr).to.equal('.');
```